src/rpcserver.cpp

```diff
@@ -321,7 +321,9 @@ static const CRPCCommand vRPCCommands[] =
     { "rawtransactions",    "getrawtransaction",      &getrawtransaction,      true,      false },
     { "rawtransactions",    "sendrawtransaction",     &sendrawtransaction,     false,     false },
     { "rawtransactions",    "signrawtransaction",     &signrawtransaction,     false,     false }, /* uses wallet if enabled */
-
+#ifdef ENABLE_WALLET
+    { "rawtransactions",    "fundrawtransaction",     &fundrawtransaction,     false,     false },
```

Comment:
Bug:  should be 'true' to indicate wallet requirement.  Otherwise, the 'pwalletMain' reference will oops when NULL (wallet support built, but disabled at runtime).